pkg/api/v1/types.go

```diff
@@ -1572,7 +1572,7 @@ type PodExecOptions struct {
 	Container string `json:"container,omitempty" description:"the container in which to execute the command. Defaults to only container if there is only one container in the pod."`
 
 	// Command is the remote command to execute; argv array; not executed within a shell.
-	Command []string `json:"command" description:"the command to execute; argv array; not executed within a shell"`
+	Command []string `json:"command,csv" description:"the command to execute; argv array; not executed within a shell"`
```

Comment:
Moved the csv commit to a different PR: https://github.com/kubernetes/kubernetes/pull/13054